sunpy/physics/differential_rotation.py

```diff
@@ -173,7 +175,8 @@ def rot_hpc(x, y, tstart, tend, frame_time='synodic', rot_type='howard', **kwarg
                                          y.to(u.arcsec).value,
```

Comment:
Also, this should **not** use `convert_hpc_hg` it should use the `sunpy.coordinates` module for the transformation.